src/shared/lib/StoreUtils.ts

```diff
@@ -287,15 +295,20 @@ export async function fetchCnaOncoKbData(sampleIdToTumorType:{[sampleId: string]
                                          discreteCNAData:MobxPromise<DiscreteCopyNumberData[]>,
                                          client: OncoKbAPI = oncokbClient)
 {
-    if (discreteCNAData.result && discreteCNAData.result.length > 0) {
+    if (!discreteCNAData.result || discreteCNAData.result.length === 0) {
```

Comment:
not important for this release, but we really have to start commenting business logic.  like, for me, relationship between discreteCNAData and oncokb is opaque.